src/server/.tests/routes/controllers/Middlewares/logInUser.spec.js

```diff
@@ -0,0 +1,54 @@
+const logInUser = require('../../../../routes/controllers/Middlewares/logInUser')
+const assert = require('chai').assert
+const sinon = require('sinon')
+
+let suite = {}
+
+describe('logInUser', () => {
+	beforeEach(() => {
+		suite = {}
+		suite.passportMock = createPassportMock()
+		suite.loginUserMock = logInUser(suite.passportMock)
+		suite.responseMock = sinon.stub()
+		suite.nextSpy = sinon.spy()
+		suite.authenticatio
+
+	})
+	it('should call next() without calling authenticationMiddleware, when req.isAuthenticated is true', () => {
```

Comment:
Give it a description from business POV, as the reason for this behavior is unclear

Reply:
ok